src/virt.c

```diff
@@ -1187,16 +1232,21 @@ static int lv_config(const char *key, const char *value) {
     return 0;
   }
   if (strcasecmp(key, "BlockDeviceFormatBasename") == 0) {
+    VALIDATE_BOOLEAN_VALUE(value, "BlockDeviceFormatBasename");
```

Comment:
Can't you use cf_util_get_boolean here?

Reply:
Initially I've tried to use 'cf_util_get_boolean' but it demands oconfig_item_t * as a parameter and it is not available from this context

Reply:
You could register lv_config as a complex config function (which I'd argue it is by now)

Reply:
>You could register lv_config as a complex config function (which I'd argue it is by now)
I had the same thoughts.

Reply:
Hi guys,
is it possible to integrate this PR as is and to send refactoring-patch for lv_config function withing next two weeks as separate PR(to change lv_config->complex config/remove probably not needed virInitialize()/refactor skipping domains logic) ?
I'm asking because before sending next patch we have to do full test cycle to be sure that we don't introduce any regressions(+some time for internal approval chain) and we are afraid that we might miss 5.9 release deadline with all bugfixes from current PR.

Reply:
>is it possible to integrate this PR as is and to send refactoring-patch for lv_config function withing next two weeks as separate PR

>>From my POV "maybe" and "in the future" are not allowed for such small changes.

I have no strong requirement to redo lv_config as a complex config function, 
but two weeks have passed and no changes published.

Reply:
Hi @rpv-tomsk, code with all required changes is already prepared (refactoring to use complex_config + removed virInitialize + refactored skipping domains + typo fixes) and we are waiting for internal green light before we can publish changes.